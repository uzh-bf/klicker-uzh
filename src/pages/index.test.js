/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Index from './index'

describe('Snapshot-Testing', () => {
  it('Works', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
