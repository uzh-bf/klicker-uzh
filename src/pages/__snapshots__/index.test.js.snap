// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot-Testing Works 1`] = `
<div
  className="jsx-1715375130 commonLayout"
>
  <div
    className="jsx-212519681 staticLayout"
  >
    <main
      className="jsx-212519681 content"
    >
      <div
        className="jsx-940135107 klicker"
      >
        <h1
          className="jsx-940135107"
        >
          Klicker
          <span
            className="jsx-940135107"
          >
            UZH
          </span>
        </h1>
        <p
          className="jsx-940135107 description"
        >
          Welcome to the open source instant audience response system.
        </p>
        <div
          className="jsx-940135107 boxes"
        >
          <div
            className="jsx-940135107 box"
          >
            <h2
              className="jsx-940135107"
            >
              Project
            </h2>
            <p
              className="jsx-940135107"
            >
              Learn more about Klicker.
            </p>
          </div>
          <div
            className="jsx-940135107 box"
          >
            <h2
              className="jsx-940135107"
            >
              Lecturer
            </h2>
            <p
              className="jsx-940135107"
            >
              How to use Klicker as a lecturer!
            </p>
          </div>
          <div
            className="jsx-940135107 box"
          >
            <h2
              className="jsx-940135107"
            >
              Audience
            </h2>
            <p
              className="jsx-940135107"
            >
              How to use Klicker as a participant.
            </p>
          </div>
          <div
            className="jsx-940135107 box"
          >
            <h2
              className="jsx-940135107"
            >
              Development
            </h2>
            <div
              className="ui divided relaxed list"
              role="list"
            >
              <div
                className="item"
                data-value={undefined}
                onClick={[Function]}
                role="listitem"
              >
                <i
                  aria-hidden="true"
                  className="github large icon middle aligned"
                />
                <div
                  className="content"
                >
                  <a
                    className="header"
                    href="https://github.com/uzh-bf/klicker-react"
                  >
                    uzh-bf/klicker-react
                  </a>
                  <div
                    className="description"
                  >
                    Frontend
                  </div>
                </div>
              </div>
              <div
                className="item"
                data-value={undefined}
                onClick={[Function]}
                role="listitem"
              >
                <i
                  aria-hidden="true"
                  className="github large icon middle aligned"
                />
                <div
                  className="content"
                >
                  <a
                    className="header"
                    href="https://github.com/uzh-bf/klicker-api"
                  >
                    uzh-bf/klicker-api
                  </a>
                  <div
                    className="description"
                  >
                    Backend (API)
                  </div>
                </div>
              </div>
              <div
                className="item"
                data-value={undefined}
                onClick={[Function]}
                role="listitem"
              >
                <i
                  aria-hidden="true"
                  className="slack large icon middle aligned"
                />
                <div
                  className="content"
                >
                  <a
                    className="header"
                  >
                    Slack-Channel
                  </a>
                  <div
                    className="description"
                  >
                    Support & Discussions
                  </div>
                </div>
              </div>
              <div
                className="item"
                data-value={undefined}
                onClick={[Function]}
                role="listitem"
              >
                <i
                  aria-hidden="true"
                  className="mail large icon middle aligned"
                />
                <div
                  className="content"
                >
                  <a
                    className="header"
                    href="mailto:support@klicker.uzh.ch"
                  >
                    support@klicker.uzh.ch
                  </a>
                  <div
                    className="description"
                  >
                    Support
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="ui list userLinks"
          role="list"
        >
          <div
            className="item"
            data-value={undefined}
            onClick={[Function]}
            role="listitem"
          >
            <a
              className="jsx-940135107"
              href="/user/login"
              onClick={[Function]}
            >
              Login
            </a>
          </div>
          <div
            className="item"
            data-value={undefined}
            onClick={[Function]}
            role="listitem"
          >
            <a
              className="jsx-940135107"
              href="/user/registration"
              onClick={[Function]}
            >
              Register
            </a>
          </div>
        </div>
      </div>
    </main>
    <footer
      className="jsx-212519681"
    >
      <hr
        className="jsx-212519681"
      />
      <p
        className="jsx-212519681"
      >
        Â©2017 IBF Teaching Center, Department of Banking and Finance, University of Zurich. All rights reserved.
        <br
          className="jsx-212519681"
        />
        Products and Services displayed herein are trademarks or registered trademarks of their respective owners.
      </p>
    </footer>
  </div>
</div>
`;
